### PR TITLE
Test cypress action

### PR DESCRIPTION
**Pull request by ddjnw1yu**
Test whether Cypress action can be triggered.

### CODE REVIEW

File: test/cypress/e2e/mapviewer.cy.js [hunk 1]
```diff
@@ -0,0 +1,92 @@
+// const datasetIds = [150]
+const datasetIds = [150, 155]
+
+// const apinatomyModels = ['Human Female']
+const apinatomyModels = ['Rat', 'Human Female']
```
Comment (alan-wu) [inline test/cypress/e2e/mapviewer.cy.js]: I think this should be named taxonModels instead.

Comment (ddjnw1yu) [reply test/cypress/e2e/mapviewer.cy.js]: Sure, will update the variable name.

File: test/cypress/support/commands.js [hunk 2]
```diff
@@ -34,4 +34,10 @@
 //       visit(originalFn: CommandOriginalFn, url: string, options: Partial<VisitOptions>): Chainable<Element>
 //     }
 //   }
-// }
\ No newline at end of file
+// }
+
+Cypress.on('uncaught:exception', (err, runnable) => {
```
Comment (alan-wu) [inline test/cypress/support/commands.js:39]: You should target specific exceptions here - we do not want to miss an exception that is avoidable - please see the following for an example - 

```
    Cypress.on('uncaught:exception', (err) => {
      // returning false here prevents Cypress from
      // failing the test
      if (err.message.includes("this.facets.at is not a function"))
        return false
      if (err.message.includes("Cannot read properties of undefined (reading 'left')"))
        return false
      return true
    })
```

**Review by alan-wu — COMMENTED** [review]
I would personally separate the pull request into two separated one for different files / pages.

**Comment by ddjnw1yu** [comment]
> I would personally separate the pull request into two separated one for different files / pages.

I will create two separate PRs once everything goes well. I merged the changes into this PR to test whether tests can pass in the cypress cloud.

**Pull request closed without merging**